docs/core/entity/media-player.md

```diff
@@ -17,43 +17,60 @@ Properties should always only return information from memory and not do I/O (lik
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
 | supported_features | int | int | Flag supported features.
-| sound_mode | string | None | The current sound mode of the media player
+| sound_mode | string | None | The current sound processing mode or filter preset of the media device
 | sound_mode_list | list | None | Dynamic list of available sound modes (set by platform, empty means sound mode not supported)
-| source | string | None | The currently selected input source for the media player.
-| source_list | list | None | The list of possible input sources for the media player. (This list should contain human readable names, suitable for frontend display)
+| source | string | None | The currently selected input source for the media device.
+| source_list | list | None | The list of possible input sources for the media device. (This list should contain human readable names, suitable for frontend display)
 | media_image_url | string | None | URL that represents the current image.
 | media_image_remotely_accessible | boolean | False | Return `True` if property `media_image_url` is accessible outside of the home network.
-| device_class | string | `None` | Type of media player.
+| device_class | string | `None` | Type of media device.
 | group_members | list | `None` | A dynamic list of player entities which are currently grouped together for synchronous playback. If the platform has a concept of defining a group leader, the leader should be the first element in that list.
+| output_list | list | None | The list of possible independent outputs for the media device e.g. a whole-house amp or matrix. (This list should contain human readable names, suitable for frontend display)
+| display_mode | string | None | The current display mode or preset of the visual media device
+| display_mode_list | list | None | Dynamic list of available display modes (set by platform, empty means sound mode not supported)
```

Comment:
This won't add them magically...

Reply:
Even if you want to add things like this @Chef-de-IT, it would mean you will be changing our entity model; which requires an architectural discussion and approval prior to creating the change.

A PR out of the blue for such a thing would be closed as well.